manual.md

```diff
@@ -12,8 +12,8 @@
 + [1.2. Faster compilation](#section1_2)
 + [1.3. Example usage](#section1_3)
 + [1.4. Dependencies](#section1_4)
-+ [2. Swizzle operators](#section2)
-+ [2.1. Standard C++98 implementation](#section2_1)
++ [2. Swizzling](#section2)
++ [2.1. Default C++98 implementation](#section2_1)
```

Comment:
Apparently this actually *is* supported, even in C++03!  Or maybe it's a compiler extension.  I'll update the manual to reflect it.